Yosemite/Yosemite/Stores/AppSettingsStore.swift

```diff
@@ -320,6 +324,24 @@ private extension AppSettingsStore {
         }
         onCompletion(numberOfDaysSinceLastDismissal >= 5)
     }
+
+    /// Sets the EU Shipping Notice dismissal state into `GeneralAppSettings`
+    ///
+    func setEUShippingNoticeDismissState(isDismissed: Bool, onCompletion: (Result<Void, Error>) -> Void) {
+        do {
+            try generalAppSettings.setValue(isDismissed, for: \.isEUShippingNoticeDismissed)
+            onCompletion(.success(()))
+        } catch {
+            onCompletion(.failure(error))
+        }
+
+    }
+
+    /// Loads the current Order Add-Ons beta feature switch state from `GeneralAppSettings`
```

Comment:
nit: Looks like this comment needs to be updated :)

Reply:
Ooops 😬 Thanks for detecting it, Rachel! 🙇 Fixed in https://github.com/woocommerce/woocommerce-ios/pull/9695/commits/59b02479e843ba3804103d57ac2788b93472034d